### ISSUE TITLE
Request to create a terrestrial mosaic function

### ISSUE THREAD

**Issue by CleverJay** [smart-underworld/seestar_alp#500]
Dear Dev team, Adding a terrestrial mosaic function would allow the user to capture Gigapixel mosaic images. It could be as simple as entering the number of vertical and horizontal tiles and amount of overlap from the current pointing position in scenery mode or to enter alt-az starting coordinates. A nice feature would also to be able to bracket the exposure when capturing. i.e. Set a plus or minus exposure value for capturing three to five exposures per tile.  Thanks for your consideration.

**Comment by xscode** [comment]
Partial duplicate of #239?

**Comment by erewhon** [comment]
This is definitely related to #239, but adds a lot of useful specifics.   And it might be easier to implement since the ask is focused on the capture.

**Comment by CleverJay** [comment]
Just to clarify, in-device stitching is not a requirement. Would only need to save out the individual tiles as images to download and stitch in other software. Thanks for your consideration.

**Comment by erewhon** [comment]
Yeah.  239 I think mentioned stitching.   Just capturing the images would be a lot more straightforward.
